### ISSUE TITLE
Standardize casing of pathogen titles

### ISSUE THREAD

**Issue by victorlin** [nextstrain/nextstrain.org#917]
_follow-up to https://github.com/nextstrain/nextstrain.org/pull/901#discussion_r1638611959_

## Context

Pathogen titles are shown in two places:

### 1. Splash page

![image](https://github.com/nextstrain/nextstrain.org/assets/13424970/c8695888-2260-4575-ac3c-fe3c61de24a1)

### 2. Core pathogens page

![image](https://github.com/nextstrain/nextstrain.org/assets/13424970/397f6bd0-7be2-4cfd-b776-a5f56d2fef68)

## Description

The titles are a mix of title case and lower case both in the context of each page (e.g. Chikungunya vs. seasonal influenza) and between the two pages (e.g. measles vs. Measles). It should be one or the other for consistency. Which one?

**Comment by victorlin** [comment]
@tsibley's vote for title case in https://github.com/nextstrain/nextstrain.org/pull/901#discussion_r1638611959, which I'm on board with:

> I also don't get why we'd want to aim for "mid-sentence usage" instead of title/heading usage since these are titles/headings after all.

**Comment by corneliusroemer** [comment]
Also the text box extends beyond the image and is bigger than necessary
<img width="431" alt="zoom us 2024-06-13 21 17 46" src="https://github.com/nextstrain/nextstrain.org/assets/25161793/6fe208f1-431e-458d-8834-ca598d0b2775">

**Comment by jameshadfield** [comment]
> Also the text box extends beyond the image

That's intentional. You may not like it, and we can of course change it, but it was a design choice I made (many moons ago now).

**Comment by ivan-aksamentov** [comment]
Another related thing that bugs me as a symmetrist is "flu" vs "influenza" vs nothing (as in in H5N1)

**Comment by victorlin** [comment]
@ivan-aksamentov I switched "flu" to "influenza" in #919. Forgot about H5N1 though. "H5N1 influenza cattle outbreak" seems a bit wordy/redundant though, I think H5N1 already implies influenza and should be sufficient?
